src/CommunicationExecutor.c

```diff
@@ -122,7 +122,8 @@ void Ros_Communication_Initialize()
     motoRosAssert(ret == RCL_RET_OK, SUBCODE_FAIL_SUPPORT_INIT);
 
     //construct a faux command line to pass to rcl_parse_arguments(..)
-    char* faux_argv[MAX_REMAP_RULE_NUM];
+    const size_t FAUX_ARGV_LEN = MAX_REMAP_RULE_NUM + 1 + 1 + 1;
```

Comment:
Not too happy with the `1 + 1 + 1` part.

We should probably refactor this at some point.